code/datums/orbit_menu.dm

```diff
@@ -0,0 +1,783 @@
+#define AB_CHECK_HANDS_BLOCKED (1<<0)
```

Comment:
?????????? why is all of action code in code/datums/orbit_menu.dm

Reply:
Probably a file copy-paste/rename error. It's not in the DME and has been deleted.